tests/test_danfoss.py

```diff
@@ -0,0 +1,241 @@
+        # SizePrefixedSimpleDescriptor(endpoint=1, profile=260, device_type=769, device_version=1, input_clusters=[0, 1, 3, 10, 32, 513, 516, 2821], output_clusters=[0, 25])
+        "endpoints": {
+            "1": {
+                "profile_id": 260,
+                "device_type": "0x0301",
+                "in_clusters": [
+                    "0x0000",
+                    "0x0001",
+                    "0x0003",
+                    "0x000a",
+                    "0x0020",
+                    "0x0201",
+                    "0x0204",
+                    "0x0b05",
+                ],
+                "out_clusters": ["0x0000", "0x0019"],
+            }
+        },
+        "manufacturer": "D5X84YU",
+        "model": "eT093WRO",
+        "class": "danfoss.thermostat.DanfossThermostat",
+    }
+
+    assert_signature_matches_quirk(
+        zhaquirks.danfoss.thermostat.DanfossThermostat, signature
+    )
+
+
+async def test_danfoss_time_bind(zigpy_device_from_quirk):
+    device = zigpy_device_from_quirk(zhaquirks.danfoss.thermostat.DanfossThermostat)
+
+    danfoss_time_cluster = device.endpoints[1].in_clusters[Time.cluster_id]
+
+    def mock_write(attributes, manufacturer=None):
+        records = [
+            WriteAttributesStatusRecord(foundation.Status.SUCCESS)
+            for attr in attributes
+        ]
+        return [records, []]
+
+    def mock_wildcard(*args, **kwargs):
+        return
+
+    patch_danfoss_trv_write = mock.patch.object(
+        danfoss_time_cluster,
+        "_write_attributes",
+        mock.AsyncMock(side_effect=mock_write),
+    )
+
+    patch_danfoss_trv_bind = mock.patch.object(
+        Time,
+        "bind",
+        mock.AsyncMock(side_effect=mock_wildcard),
+    )
+
+    with patch_danfoss_trv_bind:
+        with patch_danfoss_trv_write:
+            await danfoss_time_cluster.bind()
+
+            assert 0x0000 in danfoss_time_cluster._attr_cache
+            assert 0x0001 in danfoss_time_cluster._attr_cache
+            assert 0x0002 in danfoss_time_cluster._attr_cache
+
+
+async def test_danfoss_thermostat_write_attributes(zigpy_device_from_quirk):
+    device = zigpy_device_from_quirk(zhaquirks.danfoss.thermostat.DanfossThermostat)
+
+    danfoss_thermostat_cluster = device.endpoints[1].in_clusters[Thermostat.cluster_id]
```

Comment:
We should be able to use the `ep_attribute` here for accessing that cluster:
```suggestion
    danfoss_thermostat_cluster = device.endpoints[1].thermostat
```